src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+for (let i = 0; i < table.rows.length; i++) {
+  const row = table.rows[i];
+  const cells = row.cells[1].innerHTML;
+  const newCells = row.insertCell(row.cells.length - 1);
+
+  if (
+    row.parentNode.tagName === 'THEAD' ||
+    row.parentNode.tagName === 'TFOOT'
```

Comment:
Code Quality Issue: The condition `row.parentNode.tagName === 'THEAD' || row.parentNode.tagName === 'TFOOT'` is checking whether the parent of the current row is a thead or tfoot element. This is a bit confusing and not very future-proof. If you want to check whether the row is a header or footer row, a more robust approach would be to check whether it's not a body row, i.e. `row.parentNode.tagName !== 'TBODY'`.